taxi/tests.py

```diff
@@ -0,0 +1,2 @@
+from django.test import TestCase
```

Comment:
The file imports `TestCase` but does not define any tests. Consider adding test cases to verify the behavior of your models and views.